image-builder/image_builder.sh

```diff
@@ -264,6 +264,7 @@ setup_loop_device() {
 format_loop() {
 	local device="$1"
 	local block_size="$2"
+	local fs_type="$3"
```

Comment:
Ouch. How about also doing one of the following to make this more robust?:

- Checking the return value of `format_loop()`.
- Changing the default in the `case` `format_loop()` to be `die "Unsupported fs type: ${fs_type}"`